lib/src/package_utils.dart

```diff
@@ -33,3 +33,50 @@ String getPackageDescription(String directoryName) {
 
 String getPackageVersion(String directoryName) =>
     _getPubspec(directoryName)['version'];
+
+const String SDK_INTRO = """
```

Comment:
As in, add an argument like --intro=PATH_TO_FILE?
What really should happen is that the SDK intro file should be included in the SDK itself. That would help quite a bit.